tutorials/InterApplicationBus.subcribe.md

```diff
@@ -3,12 +3,11 @@ Subscribes to messages from the specified application on the specified topic.
 ```js
 async function subcribe(topic, listener) {
     return await fin.InterApplicationBus.subscribe({
-        uuid: 'app-1'
```

Comment:
This file has a mistake, uses incorrect markdown `#Example` instead of `# Example`